incubator/elasticsearch/values.yaml

```diff
@@ -4,12 +4,13 @@
 appVersion: "5.4"
 
 image:
-  repository: "centerforopenscience/elasticsearch"
-  tag: "5.4"
+  repository: "docker.elastic.co/elasticsearch/elasticsearch-oss"
```

Comment:
This is a cool image. 👍

Reply:
The `pre-stop-hook.sh` currently relies on the json query tool [`jq`](https://github.com/kubernetes/charts/blob/master/incubator/elasticsearch/templates/configmap.yaml#L122) to parse the k8s api responses. I'm on board with sticking with the default image, and historically had only kept the very [minimally wrapped](https://github.com/CenterForOpenScience/docker-library/blob/master/elasticsearch/5.4/Dockerfile#L5) official image due to the need for the [fabric8 plugin](https://github.com/CenterForOpenScience/docker-library/blob/master/elasticsearch/5.4/Dockerfile#L18). The script should be updated to use an inline `python` command to perform the same operations.

Reply:
A thought, but we'll likely need to wrap the image again when we get around to [adding TLS support](https://github.com/CenterForOpenScience/docker-library/blob/master/elasticsearch/5.5/tls/Dockerfile#L19), it is not included in X-Pack :-(.